backend/djangoindia/api/views/__init__.py

```diff
@@ -1,4 +1,15 @@
-from .event import EventAPIView,EventAttendeeViewSet
-from .communication import SubscriberAPIView, ContactUsAPIView
+from .communication import ContactUsAPIView, SubscriberAPIView
+from .event import EventAPIView, EventAttendeeViewSet
+from .media_library import FolderLiteSerializer, FolderSerializer
 from .partner_and_sponsor import CommunityPartnerAndSponsorAPIView
-from .media_library import FolderLiteSerializer,FolderSerializer
+
+
+__all__ = [
```

Comment:
are we utilising these direct import configurations ?

Reply:
direct imports? They are being used in the `__all__` list convention to make the module's API Public. Its a good practice.